ext/reaction_actions.py

```diff
@@ -1,11 +1,8 @@
-import lib.bing as bing
+import lib.gtranslate as gtranslate
```

Comment:
En fait tu pourrais changer comme ça, et s'il faut changer encore, le changement serait minimal
```suggestion
import lib.gtranslate as translate
```